src/main/kotlin/frc/robot/autonomous/ScoreState.kt

```diff
@@ -4,21 +4,13 @@ import edu.wpi.first.math.geometry.Pose2d
 import edu.wpi.first.wpilibj2.command.Command
 import edu.wpi.first.wpilibj2.command.Commands
 import edu.wpi.first.wpilibj2.command.Commands.runOnce
-import frc.robot.lib.distanceFromPoint
 import frc.robot.lib.flipIfNeeded
-import frc.robot.swerveDrive
+import frc.robot.subsystems.l1
 import org.littletonrobotics.junction.Logger
 
-// TODO: Add implementation for actual value
-val IS_WITHIN_AUTO_SCORE_DISTANCE = {
-    swerveDrive.pose.distanceFromPoint(
-        selectedScorePose.invoke().translation
-    ) <= MAX_ALIGNMENT_DISTANCE
-}
-
 var selectedScorePose: () -> Pose2d = { Reef1 }
 
-var isLeft = { false }
+var selectedHeightCommand: () -> Command = { l1() }
```

Comment:
It's cleaner to save the selected height in a variable, and add a command that goes to that height.